README.md

```diff
@@ -1445,6 +1445,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Patel Prabhu Teja](https://github.com/patelprabhuteja)
   - [Patricia Diaz](https://github.com/devPatricia)
   - [Patrick Reinhold Bergstroem](https://github.com/prb01)
+  - [Payal Sasmal](https://github.com/PayalSasmal10)
```

Comment:
AFAIK, your name should be in line 1449, after Paula Isabel.
Let us know if you need any help with that. 🙂

Reply:
Hi @Panquesito7, I have modified the README